CONTRIBUTING.md

```diff
@@ -122,6 +122,25 @@ $ cd www
 $ yarn start
 ```
 
+### Examples
+
+Example projects should be added in the `packages/create-sst` folder. Create a new folder in the `packages/create-sst/bin/presets` folder and then add the files for the example project you want to add. Your example can take advantage of other presets by specifying the extension in a `preset.mjs` file. 
+
+You can test your example by running the following command:
+
+```bash
+$ cd packages/create-sst
+$ yarn install
+$ node .bin/create-sst.mjs --template=examples/folder-name-for-your-new-example MyExampleApp
```

Comment:
I believe this should be 
`node ./bin/create-sst.mjs --template=examples/folder-name-for-your-new-example MyExampleApp`